lists/gt/gt-nit.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": {
+    "en": "Guatemala Tax Identification Number",
+    "local": "Número de Identificación Tributaria de Guatemala"
+  },
+  "url": "https://portal.sat.gob.gt/portal/",
+  "description": {
+    "en": "The Número de Identificación Tributaria (NIT) is a number assigned by the Superintendencia de Administración Tributaria in Guatemala to every tax payer, including both companies and individuals.\n\nCompanies must register with the Commercial Registry (Registro Mercantil) online at https://minegocio.gt/ to be incorporated and assigned an NIT number."
+  },
+  "coverage": [
+    "GT"
+  ],
+  "subnationalCoverage": [],
+  "structure": [
+    "company",
+    "trust"
+  ],
+  "sector": [],
+  "code": "GT-NIT",
+  "confirmed": false,

```

Comment:
I think you need to set confirmed to 'true' or the list won't appear in org-id

Reply:
Thanks! Fixed in https://github.com/org-id/register/pull/382/commits/ddb67c2c2a610baf625656cae5ee436a12618eb9